.gitignore

```diff
@@ -156,6 +156,17 @@ ml-20m/
 *.model
 *.mml
 nohup.out
+
+#####  kdd 2020 tutorial data folder
+scenarios/KDD2020-tutorial/data_folder.zip
```

Comment:
I think it is ok to have this, however, the lines 
```
scenarios/KDD2020-tutorial/.ipynb_checkpoints/
scenarios/academic/KDD2020-tutorial/.ipynb_checkpoints/

```
are not needed, because `.ipynb_checkpoints` is already added to the gitignore a little bit earlier. Same thing with 
```
scenarios/KDD2020-tutorial/data_folder.zip
scenarios/academic/KDD2020-tutorial/data_folder.zip
```
because `*.zip` is also defined.